api/resources_portal/config/logging.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import sys
+from multiprocessing import current_process
+
+from django.conf import settings
+
+import daiquiri
+import requests
+from retrying import retry
+
+# Found: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
+METADATA_URL = "http://169.254.169.254/latest/meta-data"
+INSTANCE_ID = None
+
+
+def get_instance_id() -> str:
+    """Returns the AWS instance id where this is running or "local"."""
+    global INSTANCE_ID
+    if INSTANCE_ID is None:
+        if settings.RUNNING_IN_CLOUD:
+
+            @retry(stop_max_attempt_number=3)
+            def retrieve_instance_id():
+                return requests.get(os.path.join(METADATA_URL, "instance-id")).text
+
+            INSTANCE_ID = retrieve_instance_id()
+        else:
+            INSTANCE_ID = "local"
+
+    return INSTANCE_ID
```

Comment:
I actually don't think we need this one in this project. We only have the one API instance so it won't help in narrowing that down like it does in refinebio.